client/src/components/projects/project-card/ProjectCard.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <div class="project-card is-clickable">
+    <div class="project-card-header is-centered is-uppercase">
+      {{ project.line }}
+    </div>
+    <div class="project-card-body">
+      <h6 class="project-card-title">
+        {{ project.name }}
+      </h6>
+      <div class="is-secondary">
+        {{ project.domain }}
+      </div>
+      <div class="project-card-description">
+        {{ project.description }}
+      </div>
+    </div>
+    <div class="project-card-footer">
+      <div class="project-card-team">
+        <img src="../../root/assets/team.svg">
+        <span>{{ project.teamcount }}</span>
+      </div>
+      <div class="is-secondary">
+        {{ project.updatedAt | date }}
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+  import Vue from 'vue';
+  import {IProject} from '../../../shared/interfaces/project';
+
+  export default Vue.extend({
+    props: {
+      project: {
+        type: IProject,
+        required: true
+      }
+    },
+    computed: {
+      teamcount() {
+        return this.project.schedules.length;
+      }
+    }
+  });
+</script>
+
+<style lang="scss" scoped>
+    @import '../../../styles/variables';
+
+    $header-height:35px;
+    $padding: 17px;
+
+    .project-card{
+        display: inline-block;
+        line-height: 1;
+        margin: 0 20px 20px 0;
+        height: 245px;
+        position: relative;
+        width:230px;
```

Comment:
Are you sure that both height and width should be locked? And this specific size? Either way extract variables

Reply:
Size of card should not change even later for mobile version. Content positioning is fixed. If description is too long it will be trimmed like that ....